WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -171,22 +171,23 @@
         </div>
       </dd>
       <% if competition.results_posted? %>
-        <dt><%= t '.highlights' %></dt>
-        <dd>
-          <div id="show_highlights">
-            <%= t('.click_to_display_highlights_html', link_here: link_to(t('common.here'), "#")) %>
-          </div>
-          <div id="hide_highlights">
-            <%= link_to(t('.hide_highlights'), "#") %>
-          </div>
-          <div class="collapse" id="highlights_text">
-            <% if main_event.present? %>
-              <p><%=md winners(competition, main_event) %></p>
-            <% end %>
-
-            <p><%=md records(competition) %></p>
-          </div>
-        </dd>
+        <% records = md records(competition) %>
+        <% main_event_message = main_event.present? ? (md winners(competition, main_event)) : "" %>
```

Comment:
Here `md` is a regular function call (it's just that confusingly enough, in ruby, you can omit parenthesis when the context allow for appropriate parsing...).
So the ternary condition could be written as:
`main_event.present? ? md(winners(competition, main_event)) : ""`
(the opening parenthesis has moved)